modules/round/src/main/History.scala

```diff
@@ -128,19 +152,19 @@ private object History {
 
   private def serverStarting = !lila.common.PlayApp.startedSinceMinutes(5)
 
-  private def load(coll: Coll, gameId: String, withPersistence: Boolean): Fu[VersionedEvents] =
+  private def load(coll: Coll, gameId: String, withPersistence: Boolean): Fu[List[VersionedEvent]] =
     coll.byId[Bdoc](gameId).map {
-      _.flatMap(_.getAs[VersionedEvents]("e")) ?? (_.reverse)
+      _.flatMap(_.getAs[List[VersionedEvent]]("e")) ?? identity
```

Comment:
is there a better way to write this?

Reply:
`doc => ~doc.flatMap(_.getAs[List[VersionedEvent]]("e"))`